solr/bin/solr

```diff
@@ -1410,13 +1415,13 @@ if [ $# -gt 0 ]; then
             SOLR_LOG_LEVEL=WARN
             shift
         ;;
-        -all)
+        --all|-all)
             stop_all=true
             shift
         ;;
-        -force)
+        --force)
```

Comment:
I talked to @epugh offline to get a better understanding of this.

My main takeaway from that discussion was that we should probably accept the single-dash *and* double-dash variant in most of these places.  Some locations don't require explicit single-dash handling, because of how the option is passed through to the Java code and handled there.  But in many cases we do, so it's probably best to handle explicit single-dash handling in most instances.

(Correct me if that's wrong @epugh )